### PR TITLE
feat:add tfsec exclude

### PR DESCRIPTION
**Pull request by Amir-Tayabali**

### CODE REVIEW

File: .github/workflows/terraform-module-validation.yml [hunk 1]
```diff
@@ -72,6 +72,11 @@ on:
         required: false
         type: string
 
+      tfsec-soft-fail:
+        default: false
+        description: "tfsec soft fai"
```
Comment (KashifSaadat) [inline .github/workflows/terraform-module-validation.yml:77]: `fai` => `fail`

Comment (gambol99) [reply .github/workflows/terraform-module-validation.yml:77]: so you want to be able to switch it off?

Comment (gambol99) [reply .github/workflows/terraform-module-validation.yml:77]: :-O

Comment (Amir-Tayabali) [reply .github/workflows/terraform-module-validation.yml:77]: lol

Comment (Amir-Tayabali) [reply .github/workflows/terraform-module-validation.yml]: No its to do with terraform-azurerm-wayfinder. Its failing as we have not set by default True for private_cluster_enabled

private_cluster_enabled             = var.private_cluster_enabled (false)


https://github.com/appvia/terraform-azurerm-wayfinder/actions/runs/9158912407/job/25178228803

Comment (gambol99) [reply .github/workflows/terraform-module-validation.yml]: add a tfsec ignore above the module definition like below [here](https://github.com/appvia/terraform-azurerm-wayfinder/blob/main/aks.tf#L1)

```hcl
# tfsec:ignore:azure-container-limit-authorized-ips
module "aks" {
  source  = "Azure/aks/azurerm"
  version = "7.3.1"
```

**Pull request closed without merging**